lib/src/io/vm.dart

```diff
@@ -37,8 +37,24 @@ bool get supportsAnsiEscapes {
   return io.stdout.supportsAnsiEscapes;
 }
 
+void print(Object? message) {
```

Comment:
Rather than overriding the default `print()` I think it's better to call this something like `safePrint()`.